core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -44,6 +59,18 @@
     @Override
     public void create() {
         pixmap = new Pixmap(WIDTH, HEIGHT, Pixmap.Format.RGBA8888);
+        stage = new Stage();
+        createBasicSkin();
+        
+        try {
+            logoTexture = new Texture(Gdx.files.internal("./core/src/de/adesso/brainysnake/img/BrainySnake_Headline.png"));
+            TextureRegion region = new TextureRegion(logoTexture, 0, 0, 629, 180);
+            logoBrainySnake = new Image(region);
+            logoBrainySnake.setPosition(Config.APPLICATION_WIDTH / 4, Config.APPLICATION_HEIGHT - 250);
+        } catch (Exception e){
```

Comment:
this is not necessary, because in the exception case, that would happen anyway

Reply:
Deleted this try/catch block.